telemetry/specs/INT.mdk

```diff
@@ -910,6 +913,190 @@ from (shim header length \* 4).
+  Domain Specific Instruction is an instruction that requires additional 
+  processing of Domain Specific ID and Domain Specific Flags. If the 
+  Domain Specific ID doesn't match the Domain ID of the node, then the transit 
+  node is required to pad the node's INT Metadata stack with the special all-ones 
+  reserved value for Domain Specific Metadata length as calculated by subtracting
+  the Baseline Metadata Length from Hop_ML.
+  If the Domain Specific ID matches the node's Domain ID then additional processing 
+  of the Domain Specific Flags is required and Domain Specific Metadata is appended
+  to the Baseline Metadata before Checksum Complement is inserted.
+  Each instruction requests 4 bytes of metadata to be inserted at each hop,
+  except if bit 6 and bit 14 is set. If bit 6 is set, the instruction requires 
+  8 bytes of metadata. If bit 14 is set, the instruction requires a domain specific
+  metadata of n bytes, n being a multiple of 4 bytes. Per-hop metadata length is 
+  set accordingly at the INT source.
+* Each INT Transit device along the path that supports INT adds its own metadata
+values as specified in the instruction bitmap immediately after the INT metadata
+header.
+  - When adding a new metadata, each device must prepend its metadata in
+    front of the metadata that are already added by the upstream devices.
+    This is similar to the push operation on a stack. Hence, the most recently
+    added metadata appears at the top of the stack. The device must add
+    metadata in the order of bits set in the instruction bitmap.
+  - If a device is unable to provide a metadata value specified in the
+    instruction bitmap because its value is not available, it must add a special
+    all-ones reserved value indicating "invalid" (4 or 8 bytes of 0xFF
+    depending on metadata length).
+  - If a device cannot add all the metadata required by the instruction bitmap
+    (irrespective of the availability of the metadata values that are asked
+    for), it must skip processing that particular INT packet entirely. This
+    ensures that each INT Transit device adds either zero bytes or
+    Per-hop Metadata Length\*4 bytes to the packet.
+  - Reserved bits in the instruction bitmap are to be handled similarly. If an
+    INT transit hop receives a reserved bit set in the instruction bitmap (e.g.
+    set by a INT source that is running a newer version), the transit hop must
+    either add corresponding metadata filled with the reserved value 0xFFFFFFFF
+    or must not add any INT metadata to the packet. This means that an
+    instruction bit marked reserved in this specification may be
+    used for a 4B metadata in a subsequent minor version while still being
+    backward compatible with this specification. However, an instruction bit
+    marked reserved in this specification may be used for a 8B metadata only
+    in the next major version, breaking backward compatibility and requring all
```

Comment:
typo: requring --> requiring
This looks to be an old typo that is present in previous versions of this spec.

Reply:
Corrected